### PR TITLE
SDIT-1128 remove unused /api/bookings/offenders/{agencyId}/list endpoint

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java [hunk 1]
```diff
@@ -425,20 +425,6 @@ public InmateDetail getBasicInmateDetail(final Long bookingId) {
         return repository.getBasicInmateDetail(bookingId).orElseThrow(EntityNotFoundException.withId(bookingId));
     }
 
-    // @VerifyAgencyAccess(overrideRoles = {"SYSTEM_USER", "VIEW_PRISONER_DATA"}) // no calls
```
Comment (karenmillermoj) [inline src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java:428]: This is the actual change - the rest is formatting

File: src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java [hunk 2]
```diff
@@ -99,22 +99,22 @@ public class InmateService {
     private final TelemetryClient telemetryClient;
 
     public InmateService(final InmateRepository repository,
-                         final CaseLoadService caseLoadService,
```
Comment (karenmillermoj) [inline src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java:102]: I didn't realise I'd reformatted it!

**Review by petergphillips — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/justice/hmpps/prison/repository/InmateRepository.java [hunk 3]
```diff
@@ -907,16 +907,6 @@ public Optional<ImprisonmentStatus> getImprisonmentStatus(final long bookingId)
         return imprisonmentStatus;
     }
 
-
-    public List<InmateBasicDetails> getBasicInmateDetailsByBookingIds(final String caseload, final List<Long> bookingIds) {
-        final var sql = InmateRepositorySql.FIND_BASIC_INMATE_DETAIL_BY_BOOKING_IDS.getSql();
```
Comment (Mjwillis) [inline src/main/java/uk/gov/justice/hmpps/prison/repository/InmateRepository.java:912]: @karenmillermoj think the SQL should go too

Comment (Mjwillis) [reply src/main/java/uk/gov/justice/hmpps/prison/repository/InmateRepository.java:912]:
```
Index: src/main/java/uk/gov/justice/hmpps/prison/repository/sql/MovementsRepositorySql.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/uk/gov/justice/hmpps/prison/repository/sql/MovementsRepositorySql.kt b/src/main/java/uk/gov/justice/hmpps/prison/repository/sql/MovementsRepositorySql.kt
--- a/src/main/java/uk/gov/justice/hmpps/prison/repository/sql/MovementsRepositorySql.kt	(revision b90f07d8e7810700e8f23994844cc189142585df)
+++ b/src/main/java/uk/gov/justice/hmpps/prison/repository/sql/MovementsRepositorySql.kt	(date 1699881359888)
@@ -23,36 +23,6 @@
     """,
   ),
 
-  GET_MOVEMENT_BY_BOOKING_AND_SEQUENCE(
-    """
-        SELECT OFFENDERS.OFFENDER_ID_DISPLAY  AS OFFENDER_NO,
-        OEM.CREATE_DATETIME            AS CREATE_DATE_TIME,
-        OEM.FROM_AGY_LOC_ID            AS FROM_AGENCY,
-        OEM.TO_AGY_LOC_ID              AS TO_AGENCY,
-        OEM.MOVEMENT_DATE,
-        OEM.MOVEMENT_TIME,
-        OEM.MOVEMENT_TYPE,
-        OEM.DIRECTION_CODE,
-        AL1.DESCRIPTION               AS FROM_AGENCY_DESCRIPTION,
-        AL2.DESCRIPTION               AS TO_AGENCY_DESCRIPTION,
-        RC1.DESCRIPTION               AS MOVEMENT_TYPE_DESCRIPTION,
-        RC2.DESCRIPTION               AS MOVEMENT_REASON,
-        RC3.DESCRIPTION               AS FROM_CITY,
-        RC4.DESCRIPTION               AS TO_CITY
-        FROM OFFENDER_EXTERNAL_MOVEMENTS OEM
-        INNER JOIN OFFENDER_BOOKINGS OB    ON OB.OFFENDER_BOOK_ID = OEM.OFFENDER_BOOK_ID
-                INNER JOIN OFFENDERS               ON OFFENDERS.OFFENDER_ID = OB.OFFENDER_ID
-        LEFT JOIN AGENCY_LOCATIONS AL1 ON OEM.FROM_AGY_LOC_ID = AL1.AGY_LOC_ID
-                LEFT JOIN AGENCY_LOCATIONS AL2 ON OEM.TO_AGY_LOC_ID = AL2.AGY_LOC_ID
-                LEFT JOIN REFERENCE_CODES RC1 ON RC1.CODE = OEM.MOVEMENT_TYPE AND RC1.DOMAIN = 'MOVE_TYPE'
-        LEFT JOIN REFERENCE_CODES RC2 ON RC2.CODE = OEM.MOVEMENT_REASON_CODE AND RC2.DOMAIN = 'MOVE_RSN'
-        LEFT JOIN REFERENCE_CODES RC3 ON RC3.CODE = OEM.FROM_CITY AND RC3.DOMAIN = 'CITY'
-        LEFT JOIN REFERENCE_CODES RC4 ON RC4.CODE = OEM.TO_CITY AND RC4.DOMAIN = 'CITY'
-        WHERE OEM.MOVEMENT_SEQ = :sequenceNumber
-        AND OEM.OFFENDER_BOOK_ID = :bookingId
-    """,
-  ),
-
   GET_MOVEMENTS_BY_OFFENDERS_AND_MOVEMENT_TYPES(
     """
         SELECT OFFENDERS.OFFENDER_ID_DISPLAY  AS OFFENDER_NO,
Index: src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateRepositorySql.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateRepositorySql.kt b/src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateRepositorySql.kt
--- a/src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateRepositorySql.kt	(revision b90f07d8e7810700e8f23994844cc189142585df)
+++ b/src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateRepositorySql.kt	(date 1699881359894)
@@ -108,26 +108,6 @@
     """,
   ),
 
-  FIND_BASIC_INMATE_DETAIL_BY_BOOKING_IDS(
-    """
-        SELECT OB.OFFENDER_BOOK_ID  as BOOKING_ID,
-        OB.BOOKING_NO,
-        OB.AGY_LOC_ID        as AGENCY_ID,
-        OB.LIVING_UNIT_ID    as ASSIGNED_LIVING_UNIT_ID,
-        I.DESCRIPTION        as ASSIGNED_LIVING_UNIT_DESC,
-        O.OFFENDER_ID_DISPLAY as OFFENDER_NO,
-        O.FIRST_NAME,
-        CONCAT(O.middle_name, CASE WHEN middle_name_2 IS NOT NULL THEN concat(' ', O.middle_name_2) ELSE '' END) MIDDLE_NAME,
-        O.LAST_NAME,
-        O.BIRTH_DATE as DATE_OF_BIRTH
-        FROM OFFENDER_BOOKINGS OB
-        INNER JOIN OFFENDERS O ON OB.OFFENDER_ID = O.OFFENDER_ID
-        LEFT JOIN AGENCY_INTERNAL_LOCATIONS I ON OB.LIVING_UNIT_ID = I.INTERNAL_LOCATION_ID
-                WHERE OB.OFFENDER_BOOK_ID IN (:bookingIds)
-        AND OB.AGY_LOC_ID = :caseloadId
-    """,
-  ),
-
   FIND_ASSIGNED_LIVING_UNIT(
     """
         SELECT B.AGY_LOC_ID,
Index: src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateAlertRepositorySql.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateAlertRepositorySql.kt b/src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateAlertRepositorySql.kt
--- a/src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateAlertRepositorySql.kt	(revision b90f07d8e7810700e8f23994844cc189142585df)
+++ b/src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateAlertRepositorySql.kt	(date 1699881359878)
@@ -86,16 +86,6 @@
     """,
   ),
 
-  GET_ALERT_CANDIDATES(
-    """
-        select distinct o.offender_id_display as offender_no
-        from OFFENDER_ALERTS icp
-        join OFFENDER_BOOKINGS ob on ob.offender_book_id = icp.offender_book_id
-        join OFFENDERS o on o.offender_id = ob.offender_id
-        where icp.modify_datetime > :cutoffTimestamp
-    """,
-  ),
-
   CREATE_ALERT(
     """
         INSERT INTO OFFENDER_ALERTS (

```

Comment (Mjwillis) [reply src/main/java/uk/gov/justice/hmpps/prison/repository/InmateRepository.java:912]: 3 SQL can go

**Pull request merged**